src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/BaselineController.kt

```diff
@@ -24,6 +31,13 @@ class BaselineController(fileManager: FileManager,
         return getAndValidate(FileType.PJNZ)
     }
 
+    @DeleteMapping("/pjnz/{hash}/")
```

Comment:
Are we requiring the hash here (unlike for the get endpoint) as a security measure (to check the front end is really deleting the file it intends to)? *Could* just look up the hash as we do for the GET.

Reply:
You're right, the hash isn't needed. Could remove?